test/ejabberd_tests/tests/ejabberdctl_SUITE.erl

```diff
@@ -595,10 +610,188 @@ stats_host(Config) ->
                 {"0\n", 0} = ejabberdctl("stats_host", ["onlineusers", SecDomain], Config)
         end).
 
+
+
+%%-----------------------------------------------------------------
+%% Improve coverage
+%%-----------------------------------------------------------------
+
+
+
+simple_register(Config) ->
+    %% given
+    {_, Domain, _} = get_user_data(kate, Config),
+    {Name, Password} = {<<"tyler">>, <<"durden">>},
+    %% when
+    {_, _} = ejabberdctl("register", [Name, Domain, Password], Config),
+    {R2, _} = ejabberdctl("registered_users", [Domain], Config),
+    %% then
+    {match, _} = re:run(R2, ".*(" ++binary_to_list(Name)++").*").
+
+simple_unregister(Config) ->
+    %% given
+    {_, Domain, _} = get_user_data(kate, Config),
+    {Name, _} = {<<"tyler">>, <<"durden">>},
+    %% when
+    {_, _} = ejabberdctl("unregister", [Name, Domain], Config),
+    {R2, _} = ejabberdctl("registered_users", [Domain], Config),
+    %% then
+    nomatch = re:run(R2, ".*(" ++binary_to_list(Name)++").*").
+
+register_twice(Config) ->
+    %% given
+    {_, Domain, _} = get_user_data(kate, Config),
+    {Name,  Password} = {<<"tyler">>, <<"durden">>},
+    %% when
+    {_, _} = ejabberdctl("register", [Name, Domain, Password], Config),
+    {R, _} = ejabberdctl("register", [Name, Domain, Password], Config),
+    %% then
+    {match, _} = re:run(R, ".*(already registered).*"),
+    {_, _} = ejabberdctl("unregister", [Name, Domain], Config).
+
+%% Fix it
+set_master_node(Config) ->
+    {R, _} = ejabberdctl("set_master", ["self"], Config),
+    nomatch = re:run(R, ".+").
```

Comment:
It's actually not so easy  to do this test in this SUITE. In case to test correctly `set_master` command we need  more distributed erlang nodes runing. I will try to implement this testcase in `cluster_commands_SUITE` module.